### PR TITLE
Polish availibility settings

### LINKED ISSUE #29319: CalDAV availibility settings design polishing
We should polish the design of the availability settings from https://github.com/nextcloud/server/issues/27422 / https://github.com/nextcloud/server/pull/27466 before the release of 23

This is what it looks like right now

Bildschirmfoto von 2021-10-19 11-52-36

* Delete/Add buttons not aligned with the time pickers
* Text between time pickers has unbalanced white space left/right
* "Please select a time zone:" should possibly lose the ":"

@GretaD @nimishavijay may I ask you two to look into this? You've worked on the first version :)

### PR DESCRIPTION
**Pull request by GretaD**
fixes #29319

### CODE REVIEW

**Comment by GretaD** [comment]
sha d1c984e it looks like this:

AVV_SETTIGS

Im failing to understand why i cannot style add slot button. It is shown correctly only when we have 2 slots like on Friday.

**Comment by ChristophWurst** [comment]
> Im failing to understand why i cannot style add slot button. It is shown correctly only when we have 2 slots like on Friday.

Where is it shown incorrectly?

**Comment by GretaD** [comment]
> > Im failing to understand why i cannot style add slot button. It is shown correctly only when we have 2 slots like on Friday.
> 
> Where is it shown incorrectly?

All the other days is positioned a bit down. When we have one slot, or no slots.

**Comment by GretaD** [comment]
availabilitysettings

File: apps/dav/src/views/Availability.vue [hunk 1]
```diff
@@ -37,16 +41,20 @@
...
-						class="add-another button"
-						@click="addSlot(day)">
-						{{ $t('dav', 'Add slot') }}
-					</button>
+					<span v-if="day.slots.length === 0"
+						class="empty-content">
+						{{ $t('dav', 'No working hours set') }}
+					</span>
 				</div>
+				<button
```
Comment (ChristophWurst) [inline apps/dav/src/views/Availability.vue:49]:
```suggestion
				<button :key="`add-slot-${day.id}`"
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Review by st3iny — APPROVED** [review]

**Pull request merged**